anndata/base.py

```diff
@@ -208,9 +208,6 @@ def _fix_shapes(X, single_col=False):
     elif len(X.shape) == 1:
         n_vars = X.shape[0]
         n_obs = 1
-    elif len(X.shape) == 1 and single_col:
```

Comment:
why don't we do flattening anymore?

Reply:
if this doesn't work anymore, it will break a lot of code... are you here at the institute?

Reply:
i’m not. and this argument wasn’t actually used, that’s why i removed it. you could find out why it wasn’t and fix it if that’s a bug.

Reply:
yes, it's a bug that I seem to have removed it... it should have been passed to `_fix_shapes`, otherwise, after slicing a single column, while the following statement will still hold true,
```
adata[:, 0].X == adata.X[:, 0]
```
the left hand side object should give weird effects... don't you agree? I'm astonished that there is no test that fails...

but fine, I could possibly also restore a proper behavior after merging...

Reply:
OK, great! that’s something pycharm caught: it highlighted the param in grey and said “isn’t used”.